### ISSUE TITLE
Billing Agreement setup_fee always return cancelled state

### ISSUE THREAD

**Issue by jiteshtandel** [paypal/PayPal-PHP-SDK#1257]
Hi, 

I wanted to charge a user immediately, so I configured **setup_fee** along with **initial_fail_amount_action=CANCEL** as per mentioned in the thread #504 
I want to charge users a monthly. To do so, I set **start_date** offset to be the next month from the the current date. 

Following are the configuration I applied first.
* frequency=month
* frequency_interval=1
* type=REGULAR

* setup_fee=Euro 10.00
* auto_bill_amount=yes
* initial_fail_amount_action=CANCEL
* max_fail_attempts=0
* start_date = date("c", strtotime("+1 month"))

However, I have enough balance in my card. The PayPal always failed to paid initial amount and returns with "**Cancelled** " status after "**Execute Agreement**" API CALL. While If I set below configuration, API returns "Active" and I can also see the amount of Euro 10.00 deposited into the merchant account with the same card.
* frequency=month
* frequency_interval=1
* type=REGULAR

* setup_fee=Euro 10.00
* auto_bill_amount=yes
* initial_fail_amount_action=CONTINUE
* max_fail_attempts=0
* start_date = date("c", strtotime("+5 minutes"))

I am facing this issue in the Production environment. I don't know what's the reason. How can we issue this issue?
Basically, I need immidate response either Cancelled or Active.Based on Active or Cancelled status, I can display proper response to the user.

Please help. Let me know if you want to know more.

Thanks in Advance.

**Comment by sucithrashanu** [comment]
@jiteshtandel 
Can you please share us the PayPal-Debug-IDs to check further?

Thanks

**Comment by jiteshtandel** [comment]
Check this link to my Payment Configuration for billing subscription.

https://github.com/paypal/PayPal-PHP-SDK/issues/504#issuecomment-586575729
